app/api/custom/orders.py

```diff
@@ -32,36 +32,35 @@
 ticket_blueprint = Blueprint('ticket_blueprint', __name__, url_prefix='/v1/tickets')
 
 
-@ticket_blueprint.route('/<string:order_identifier>')
-@order_blueprint.route('/<string:order_identifier>/tickets-pdf')
+@ticket_blueprint.route('/<int:attendee_id>')
+@order_blueprint.route('/<int:attendee_id>/tickets-pdf')
 @jwt_required
-def ticket_attendee_authorized(order_identifier):
+def ticket_attendee_authorized(attendee_id):
     if current_user:
         try:
-            order = Order.query.filter_by(identifier=order_identifier).first()
+            ticket_holder = TicketHolder.query.filter_by(id=attendee_id).first()
+            order = Order.query.filter_by(id=ticket_holder.order_id).first()
         except NoResultFound:
-            raise NotFoundError(
-                {'source': ''}, 'This ticket is not associated with any order'
-            )
+            raise NotFoundError({'source': ''}, 'No Ticket found for attendee')
         if (
             has_access(
                 'is_coorganizer_or_user_itself',
-                event_id=order.event_id,
-                user_id=order.user_id,
+                event_id=ticket_holder.event_id,
+                user_id=ticket_holder.user.id,
             )
             or order.is_attendee(current_user)
         ):
-            key = UPLOAD_PATHS['pdf']['tickets_all'].format(identifier=order_identifier)
+            key = UPLOAD_PATHS['pdf']['tickets_all'].format(identifier=order.identifier)
```

Comment:
This is taking order identifier as key, how can it ever be for an attendee. Please understand the issue first

Reply:
Can I know the path of the attendee pdf?